web/modules/custom/bnf/bnf.module

```diff
@@ -20,10 +20,15 @@ function bnf_entity_base_field_info(EntityTypeInterface $entity_type): array {
   if ($entity_type->id() === 'node') {
     $fields[BnfStateEnum::FIELD_NAME] = BaseFieldDefinition::create('enum_integer')
       ->setName(BnfStateEnum::FIELD_NAME)
-      ->setLabel(t('BNF State'))
-      ->setDescription(t('The BNF state of the entity, defining if it was imported, exported, or neither.'))
+      ->setLabel(t('BNF State', [], ['context' => 'BNF']))
+      ->setDescription(t('The BNF state of the entity, defining if it was imported, exported, or neither.', [], ['context' => 'BNF']))
       ->setSetting('enum_class', BnfStateEnum::class)
       ->setDefaultValue(BnfStateEnum::None);
+
+    $fields['bnf_source_name'] = BaseFieldDefinition::create('string')
```

Comment:
You need to tell the entity system that the entity has been updated. See https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/64d2c73b9f43e6be79609959febf8a9f3f7188bc/web/modules/custom/bnf/bnf_client/bnf_client.install#L19

Oh, and this is instead of the canonical title? I think I get what you mean now, but I'm not entirely sure if we need to show where the content originated?

Reply:
We do need to show it, it is part of the assignment - it needs to be shown on /content on the server side :) I don't think it needs to be passed along from server to client though.